image/autodetect.go

```diff
@@ -34,6 +36,13 @@ const (
 	TypeConfig       = "config"
 )
 
+// Layoutfile realize the image files that are included in the image layout type
+var Layoutfile = []string{
+	"blobs",
+	"oci-layout",
+	"refs",
```

Comment:
this is changed now. There is no longer a ./refs/ directory. And there is an `index.json` file. See the updated image-layout.md doc.